node/test/@opent2t/test-b/TranslatorBC.js

```diff
@@ -1,78 +0,0 @@
-"use strict";
-// This code uses ES2015 syntax that requires at least Node.js v4.
-// For Node.js ES2015 support details, reference http://node.green/
-
-const EventEmitter = require("events");
-
-/**
- * This translator class implements two interfaces: "org.opent2t.test.B"
- * and "org.opent2t.test.C". Since those interfaces do not have any conflicting
- * member names, they are both implemented directly.
- */
-class TestTranslatorBC extends EventEmitter {
-
-    constructor(deviceProps) {
```

Comment:
Thanks... yup I understand this is deleted code it just jogged my memory and I was intending to say let's keep constructor pattern... makes my code (in my PR) easier to read.

@sanjaiganesh :shipit: please merge.